backend/clubs/views.py

```diff
@@ -2058,6 +2058,70 @@ def perform_destroy(self, instance):
             context=context,
         )
 
+    @action(detail=False, methods=["GET"])
```

Comment:
Can we make this an instance-specific route? Ideally, it will be accessed when OSA is viewing a specific club's page and wants to see the diff between the latest and latest approved versions.